content/collective_intro.tex

```diff
@@ -47,3 +91,17 @@
 \item \FUNC{shmem\_alltoall\{32, 64\}}
 \item \FUNC{shmem\_alltoalls\{32, 64\}}
 \end{itemize}
+\end{DeprecateBlock}
+
+\newtext{
+Additionally, \openshmem provides two synchronization routines for backward
+compatibility, which are defined with semantic equivalence to team- and
+context-based routines:
```

Comment:
If this text is to be added in this PR, should we also go ahead and change the individual descriptions of these routines in the same PR?

Reply:
Ah, good point. I'll have to make those changes, too.